src/messages/responses/set-fetch.ts

```diff
@@ -0,0 +1,37 @@
+import {ResponseBase} from './response-base';
```

Comment:
Do we want to name this file `cache-set-fetch` to align with the .NET SDK?

Reply:
Ummmm.... yes... good callout.